db/models.py

```diff
@@ -1 +1,29 @@
+from typing import ForwardRef
+
 from django.db import models
+from django.db.models import SET_NULL, CASCADE
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField()
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=CASCADE)
```

Comment:
Consider adding a `related_name` attribute to the ForeignKey field for better access to related objects. This is missing in the `Skill` model.